webpack/assets/javascripts/react_app/components/hosts/storage/vmware/StorageContainer.test.js

```diff
@@ -51,4 +51,18 @@ describe('StorageContainer', () => {
 
     expect(wrapper.render().find('.controller-container')).toHaveLength(0);
   });
+
+  it('changes controller type when one is selected', () => {
```

Comment:
it seems that you can use our testHelpers instead of mounting components manually.
we have already test helpers for component [unit-test](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/testHelpers.js) and [integration tests](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/common/IntegrationTestHelper.js) with redux's actions and reducers.